test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -902,30 +898,30 @@ def revert_pbsconf(self):
 
         # Revert server hosts' pbs.conf
         for server in self.servers.values():
-            default_vals = none_vals
+            default_vals = dict(none_vals)
             cmds_to_exec = []
             dmns_to_start = 0
-            pbs_conf_filepath = self.du.get_pbs_conf_file(server.hostname)
-            pbs_conf_val = self.du.parse_pbs_config(server.hostname,
-                                                    pbs_conf_filepath)
-            if not pbs_conf_val or len(pbs_conf_val) < 1:
+            self.du.get_pbs_conf_file(server.hostname)
```

Comment:
Why are you calling this?  It returns the conf file name.  By calling it like this, it does nothing.  parse_pbs_config() will call it itself.